purchases/build.gradle

```diff
@@ -81,18 +79,18 @@ dependencies {
     testImplementation libs.okhttp.mockwebserver
     testImplementation libs.playServices.ads.identifier
 
-    integrationTestImplementation libs.androidx.appcompat
-    integrationTestImplementation libs.material
-    integrationTestImplementation libs.androidx.constraintlayout
-
-    androidTestIntegrationTestImplementation libs.androidx.lifecycle.runtime.ktx
-    androidTestIntegrationTestImplementation libs.androidx.test.espresso.core
-    androidTestIntegrationTestImplementation libs.androidx.test.runner
-    androidTestIntegrationTestImplementation libs.androidx.test.rules
-    androidTestIntegrationTestImplementation libs.androidx.test.junit
-    androidTestIntegrationTestImplementation libs.assertJ
-    androidTestIntegrationTestImplementation libs.mockk.android
-    androidTestIntegrationTestImplementation libs.mockk.agent
+    androidTestImplementation libs.androidx.appcompat
+    androidTestImplementation libs.androidx.core
+    androidTestImplementation libs.androidx.lifecycle.common
+    androidTestImplementation libs.androidx.lifecycle.runtime.ktx
+    androidTestImplementation libs.androidx.lifecycle.process
```

Comment:
There are some new dependencies here, why are they needed?

Reply:
Oh my bad, I was testing some things around why `AppCompatActivity` wasn't working and I forgot to remove those. Will remove those. Good catch!